src/app/zap-templates/zcl/data-model/chip/ecosystem-information-cluster.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="General"/>
+  <struct name="EcosystemDeviceStruct" isFabricScoped="true" apiMaturity="provisional">
```

Comment:
why EcosystemDeviceStruct is FabricScoped?

Reply:
See https://github.com/project-chip/connectedhomeip/pull/34291/files#r1674036642 on why it is Fabric scoped